src/DatePicker/stories/index.stories.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { ComponentStory, ComponentMeta } from '@storybook/react'
+import { Form, FormItemProps } from 'antd'
+import moment from 'moment'
```

Comment:
moment is not maintained anymore. Can use date-fns

Reply:
default type of the Date Picker props : disabledDate is Moment.